starter/source/materials/tools/nlsqf.F

```diff
@@ -1385,4 +1386,538 @@ REAL FUNCTION RAND1(IVAL)
       
       RETURN
       END FUNCTION RAND1
+Chd|====================================================================
```

Comment:
Hi @SaidMamouri 
can you put this new routine in a separate source file ?

Thanks